.gitmodules

```diff
@@ -1,9 +1,12 @@
 [submodule "google-benchmark"]
 	path = ext/google-benchmark
 	url = https://github.com/google/benchmark
+    branch = main
```

Comment:
whitespace error here and below

Reply:
What's the error? This file is generated, and I would assume `git` knows how ;)